docs/skopeo.1.md

```diff
@@ -18,6 +18,10 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   default read from `$HOME/.kube/config`, which is set e.g. using
   `(oc login)`.
```

Comment:
Changes to match your comments although added OpenShift(Atomic) since we now refer to this as the OpenShift Registry.  We might want to change the name atomic to openshift and support both.